RibonDAO/interface
# Tech 479: Show only non profits with pool balance

# Description

- Show only non profits and causes with pool balance on causesPage

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Create a pool balance with balance 0 and see the home page changing
